### PR TITLE
Revertconcrurency

### PR DESCRIPTION
**Pull request by minborg**
I have rolled back the concurrency changes I made and reverted back to the old solution. This prevents tests from being stuck on Linux.

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/jlbh/JLBH.java [hunk 1]
```diff
@@ -67,11 +67,11 @@
     private final Histogram endToEndHistogram = createHistogram();
     @NotNull
     private final Histogram osJitterHistogram = createHistogram();
-    private final AtomicLong noResultsReturned = new AtomicLong();
+    private volatile long noResultsReturned;
```
Comment (RobAustin) [inline src/main/java/net/openhft/chronicle/jlbh/JLBH.java:75]: why is this "volatile"

Comment (RobAustin) [reply src/main/java/net/openhft/chronicle/jlbh/JLBH.java:75]: if this is not thread safe ?

**Review by RobAustin — CHANGES_REQUESTED** [review]
please document that this library IS NOT thread-safe!

File: src/main/java/net/openhft/chronicle/jlbh/JLBH.java (hunk 1 above)
Comment (minborg) [reply src/main/java/net/openhft/chronicle/jlbh/JLBH.java:75]: No one knows...

Comment (minborg) [reply src/main/java/net/openhft/chronicle/jlbh/JLBH.java:75]: Perhaps best to leave it as it is?

**Review by alamar — APPROVED** [review]

**Review by RobAustin — APPROVED** [review]

**Pull request merged**